File: simple/csharp/extract-variable_after2.cs
void renderBanner() 
{
  bool isChanged = frame.isChanged || target.isChanged;
  bool mustRedraw = isChanged || experiment.isRunning();
  bool isFullScreen = frame.getSize() == screen.getSize();

  if (isFullScreen && mustRedraw) 
  {
    // print banner
  }
}

File: simple/java/extract-variable_after2.java
void renderBanner() {
  boolean isChanged = frame.isChanged || target.isChanged;
  boolean mustRedraw = isChanged || experiment.isRunning();
  boolean isFullScreen = frame.getSize() == screen.getSize();

  if (isFullScreen && mustRedraw) {
    // print banner
  }
}
